pages/common/go.md

```diff
@@ -0,0 +1,23 @@
+# go
+
+> Tool for managing go source code.
+
+- Download and install a package, specified by its import path:
+
+`go get {{package_path}}`
+
+- Compile and run a source file (it has to contain a "main" package):
+
+`go run {{file}}.go`
+
+- Compile the package present in the current directory:
+
+`go build`
+
+- Execute all test cases of the current package (files have to end with "_test.go"):
```

Comment:
I am thinking it might be a little inconsistent because I have main in "..." a few lines above, now this will be in `...`. Anyway, this is just an issue with github rendering, the client shows it fine. What do you think ?